packages/@react-spectrum/list/test/ListView.test.js

```diff
@@ -624,4 +625,436 @@ describe('ListView', function () {
       expect(grid.scrollTop).toBe(0);
     });
   });
+
+  describe('drag and drop', function () {
+    function DraggableListView(props) {
+      let {dragHookOptions, listViewProps} = props;
+      return (
+        <>
+          <Droppable onDrop={onDrop} />
+          <DragExample dragHookOptions={{onDragStart, onDragMove, onDragEnd, ...dragHookOptions}} listViewProps={{onSelectionChange, ...listViewProps}} />
+        </>
+      );
+    }
+    beforeEach(() => {
+      jest.spyOn(HTMLElement.prototype, 'getBoundingClientRect').mockImplementation(() => ({
+        left: 0,
+        top: 0,
+        x: 0,
+        y: 0,
+        width: 100,
+        height: 50
+      }));
+    });
+
+    afterEach(() => {
+      act(() => {jest.runAllTimers();});
+      jest.clearAllMocks();
+    });
+
+    describe('via mouse', function () {
+      it('should show a default drag preview on drag', function () {
+        let {getAllByRole, getByTestId} = render(
+          <DraggableListView />
+        );
+
+        let row = getAllByRole('row')[0];
+        let cell = within(row).getByRole('gridcell');
+
+        // Need raf to be async so the drag preview shows up properly
+        jest.spyOn(window, 'requestAnimationFrame').mockImplementation(cb => setTimeout(cb, 0));
+        let dataTransfer = new DataTransfer();
+
+        fireEvent(cell, new DragEvent('dragstart', {dataTransfer, clientX: 5, clientY: 5}));
```

Comment:
odd, the dataTransfer goes through but the clientX and Y are undefined when I use this...

Reply:
see if they have a bug? if they don't we can report one

Reply:
Will try bumping the versions of testing library and jsdom in a bit, will need to narrow down which library is responsible for this (think it might be jsdom from some of the issues I dug up)

Reply:
if it is something that requires a version bump of the library, maybe don't worry about it, or open a separate PR for it so it's not part of this